test/image/test_vector_image.py

```diff
@@ -192,3 +187,46 @@ def test_svg_image_billion_laughs():
                 b"<lolz>&lol9;</lolz>"
             )
         )
+
+
+def test_svg_image_fit_rect(tmp_path):
+    """
+    Scale the image to fill the rectangle, keeping its aspect ratio,
+    and ensure it does overflow the rectangle width or height in the process.
+    """
+    pdf = fpdf.FPDF()
+    pdf.add_page()
+    test_file = SVG_SRCDIR / "SVG_logo.svg"
+
+    rect1 = 30, 30, 60, 100
+    pdf.rect(*rect1)
+    pdf.image(test_file, *rect1, keep_aspect_ratio=True)
+
+    rect2 = 100, 30, 100, 60
+    pdf.rect(*rect2)
+    pdf.image(test_file, *rect2, keep_aspect_ratio=True)
+
+    assert_pdf_equal(pdf, HERE / "svg_image_fit_rect.pdf", tmp_path)
+
+
+IMG_DESCRIPTION = "The Mighty SVG Logo"
+
+
+def test_svg_image_altt_title(tmp_path):
+    test_file = SVG_SRCDIR / "SVG_logo.svg"
+    pdf = fpdf.FPDF()
+    pdf.add_page()
+    pdf.image(test_file, alt_text=IMG_DESCRIPTION, w=50, h=50)
+    pdf.image(test_file, title=IMG_DESCRIPTION, w=50, h=50)
+    pdf.image(test_file, alt_text=IMG_DESCRIPTION, title=IMG_DESCRIPTION, w=50, h=50)
+    assert_pdf_equal(pdf, HERE / "svg_image_altt_title.pdf", tmp_path)
+
+
+def test_svg_image_altt_two_pages(tmp_path):
```

Comment:
I think there is a typo in those 2 unit tests: `altt` -> `alt`

Reply:
"altt" ~ "alt text".

There are quite a few tests with names that are way too long and get truncated in the output, making it impossible to tell different tests apart.
Because of that, I'm trying to keep the names concise. The task is to make a specific test identifyable, not to produce literature...